science/imager/Portfile

```diff
@@ -22,13 +22,13 @@ long_description    IMAGER (and its associated visualization tool VIEWER) is a s
                     and the ALMA or JVLA arrays, with a simple, convenient user interface.
 homepage            https://imager.oasu.u-bordeaux.fr
 
-set site_base       https://nuage.oasu.u-bordeaux.fr/index.php/s/PoxdxMTTYXqNwKF
+set site_base       https://sdrive.cnrs.fr/s/L5ppFWpkkX7yYSr
 master_sites        ${site_base}/download?path=/&files=
 distname            patched-macport-${name}-${my_version}
 
-checksums           rmd160  9f09c675e43492261f196c7de11a8efb252ee736 \
-                    sha256  4187c417c841ee010910948f4751d92ce0e2a187cbfb856760a2eb377dae2b99 \
-                    size    32496624
+checksums           rmd160  7b36e6706c3b46f05ca2b93c2eab2b89b5699736 \
+                    sha256  bb9a240b83f4a2bd5036a2ac77f0bfa33443512ecefe47393551032bb2c93269 \
+                    size    32602225
 
 set py_ver          3.11
```

Comment:
this should use `3.12` if possible; that has been the default Python version in MacPorts since the beginning of this year.